ElectoralRegisterResidentInformationApi.Tests/V1/Gateways/ElectoralRegisterResidentInformationApiGatewayTests.cs

```diff
@@ -1,24 +1,24 @@
 using AutoFixture;
-using BaseApi.Tests.V1.Helper;
-using BaseApi.V1.Domain;
-using BaseApi.V1.Gateways;
+using ElectoralRegisterResidentInformationApi.Tests.V1.Helper;
+using ElectoralRegisterResidentInformationApi.V1.Domain;
+using ElectoralRegisterResidentInformationApi.V1.Gateways;
 using FluentAssertions;
 using NUnit.Framework;
 
-namespace BaseApi.Tests.V1.Gateways
+namespace ElectoralRegisterResidentInformationApi.Tests.V1.Gateways
 {
     //TODO: Rename Tests to match gateway name
     //For instruction on how to run tests please see the wiki: https://github.com/LBHackney-IT/lbh-base-api/wiki/Running-the-test-suite.
     [TestFixture]
-    public class ExampleGatewayTests : DatabaseTests
+    public class ElectoralRegisterResidentInformationApiGatewayTests : DatabaseTests
     {
         private readonly Fixture _fixture = new Fixture();
-        private ExampleGateway _classUnderTest;
+        private ElectoralRegisterResidentInformationApiGateway _classUnderTest;
```

Comment:
Could we rename the gateways without API in the name:

e.g. `ElectoralRegisterGateway`